rebench/persistence.py

```diff
@@ -351,6 +354,7 @@ def run_completed(self):
     def _open_file_to_add_new_data(self):
         if not self._file:
             self._file = self._open_file_and_append_execution_comment()
+            self._persists_data_titles_in_open_file()
```

Comment:
Was there a specific reason not to do this inside of `self._open_file_and_append_execution_comment()` as part of the construction of what's called the `shebang_line`? This should probably also renamed, since it's the shebang line and additional metadata.